src/components/hv-picker-field/index.ios.js

```diff
@@ -0,0 +1,225 @@
+// @flow
```

Comment:
should this file be removed and/or its content merged to the sibling index.ios.tsx?